src/refract.c

```diff
@@ -151,6 +153,7 @@ double novas_optical_refraction(double jd_tt, const on_surface *loc, enum novas_
  * @sa on_surface
  */
 double novas_radio_refraction(double jd_tt, const on_surface *loc, enum novas_refraction_type type, double el) {
+  (void)jd_tt;
```

Comment:
I like to add a space after the cast style-wise, i.e. `(void) jd_tt;`

Reply:
Do you have a suggested overall C style such that these kind of edits can be a bit more automatic (say with clang-format)?

Reply:
Haha, not really. It's just how I like things. I have configured Eclipse with my preferences, which are committed to the repo. I'm generally not too picky, except that I like to be consistent one way or another. The Eclipse formatter works 95% of the time, but it does screw up here an there and needs a little careful manual checking for SNAFUs after...